### ISSUE TITLE
Error when tab-complete or execute invalid boolean argument

### ISSUE THREAD

**Issue by BlackBaroness** [Rollczi/LiteCommands#205]
Error:
```java
[15:10:36 ERROR]: Exception when BlackBaroness attempted to tab complete comet seer fa
org.bukkit.command.CommandException: Unhandled exception executing tab-completer for 'comet seer fa' in ru.elementcraft.elementcomets.__relocated__.litecommands.bukkit.SimpleCommand(comet)
        at org.bukkit.command.SimpleCommandMap.tabComplete(:237) ~[guardspigot.jar:git-Paper-8359863]
        at org.bukkit.command.SimpleCommandMap.tabComplete(:183) ~[guardspigot.jar:git-Paper-8359863]
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.tabCompleteCommand(:1760) ~[guardspigot.jar:git-Paper-8359863]
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.tabComplete(:1734) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.MinecraftServer.tabCompleteCommand(:1279) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.PlayerConnection$9.evaluate(:2345) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.PlayerConnection$9.evaluate(:2342) ~[guardspigot.jar:git-Paper-8359863]
        at org.bukkit.craftbukkit.v1_12_R1.util.Waitable.run(:24) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(:865) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(:431) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(:778) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(:669) ~[guardspigot.jar:git-Paper-8359863]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.IllegalArgumentException: Argument is not boolean
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.BooleanArgument.lambda$parse$2(BooleanArgument.java:15) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.panda_std.Option.orThrow(Option.java:244) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.BooleanArgument.parse(BooleanArgument.java:15) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.TypeUtils.validate(TypeUtils.java:46) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.TypeUtils.lambda$suggestion$1(TypeUtils.java:37) ~[?:?]
        at java.util.Optional.filter(Optional.java:218) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.TypeUtils.suggestion(TypeUtils.java:37) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.AbstractBasicTypeArgument.suggest(AbstractBasicTypeArgument.java:30) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.simple.SimpleMultilevelArgument.suggestion(SimpleMultilevelArgument.java:51) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.simple.SimpleMultilevelArgument.suggestion(SimpleMultilevelArgument.java:16) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterImpl.extractSuggestion(AnnotatedParameterImpl.java:39) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterImpl$SimpleSuggester.suggestion(AnnotatedParameterImpl.java:102) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.suggestion.Suggester.extractSuggestions(Suggester.java:18) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.suggestionParameters(LiteCommandSection.java:183) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.findSuggestion(LiteCommandSection.java:145) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.findSuggestion(LiteCommandSection.java:131) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.command.CommandService.lambda$register$1(CommandService.java:47) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.bukkit.SimpleCommand.tabComplete(SimpleCommand.java:35) ~[?:?]
        at org.bukkit.command.Command.tabComplete(:90) ~[guardspigot.jar:git-Paper-8359863]
        at org.bukkit.command.SimpleCommandMap.tabComplete(:231) ~[guardspigot.jar:git-Paper-8359863]
        ... 12 more
```

When you try to write a command like "[command] fa" instead of "[command] false", you will receive that error.

```java
    @Async
    @Execute(route = "seer")
    public void seer(Player player, @Arg boolean enable) {
    }
```

Also executing command with non-boolean value throws error too:
```java
[15:14:24 INFO]: BlackBaroness issued server command: /comet seer fa
[15:14:24 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'comet seer fa' in ru.elementcraft.elementcomets.__relocated__.litecommands.bukkit.SimpleCommand(comet)
        at org.bukkit.command.SimpleCommandMap.dispatch(:162) ~[guardspigot.jar:git-Paper-8359863]
        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(:687) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(:1504) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.PlayerConnection.a(:1309) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(:45) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(:5) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(:14) ~[guardspigot.jar:git-Paper-8359863]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
        at net.minecraft.server.v1_12_R1.SystemUtils.a(:46) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(:854) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(:431) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(:778) ~[guardspigot.jar:git-Paper-8359863]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(:669) ~[guardspigot.jar:git-Paper-8359863]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: ru.elementcraft.elementcomets.__relocated__.panda_std.AttemptFailedException: java.lang.IllegalArgumentException: Argument is not boolean
        at ru.elementcraft.elementcomets.__relocated__.panda_std.Result.supplyThrowing(Result.java:140) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.TypeUtils.parse(TypeUtils.java:29) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.AbstractBasicTypeArgument.parse(AbstractBasicTypeArgument.java:25) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.simple.OneArgument.parseMultilevel(OneArgument.java:21) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.simple.SimpleMultilevelArgument.match(SimpleMultilevelArgument.java:34) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterImpl.match(AnnotatedParameterImpl.java:35) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterStateImpl.lambda$new$0(AnnotatedParameterStateImpl.java:20) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.HandleLazyUtil.lambda$handle$0(HandleLazyUtil.java:16) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.panda_std.Lazy.get(Lazy.java:50) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterStateImpl.matchResult(AnnotatedParameterStateImpl.java:26) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteArgumentArgumentExecutor.find(LiteArgumentArgumentExecutor.java:78) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.find(LiteCommandSection.java:244) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.find(LiteCommandSection.java:225) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.execute(LiteCommandSection.java:83) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.command.CommandService.lambda$register$0(CommandService.java:42) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.bukkit.SimpleCommand.execute(SimpleCommand.java:29) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(:152) ~[guardspigot.jar:git-Paper-8359863]
        ... 14 more
Caused by: java.lang.IllegalArgumentException: Argument is not boolean
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.BooleanArgument.lambda$parse$2(BooleanArgument.java:15) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.panda_std.Option.orThrow(Option.java:244) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.BooleanArgument.parse(BooleanArgument.java:15) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.AbstractBasicTypeArgument.lambda$parse$0(AbstractBasicTypeArgument.java:25) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.panda_std.Result.supplyThrowing(Result.java:133) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.TypeUtils.parse(TypeUtils.java:29) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.basictype.AbstractBasicTypeArgument.parse(AbstractBasicTypeArgument.java:25) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.simple.OneArgument.parseMultilevel(OneArgument.java:21) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.argument.simple.SimpleMultilevelArgument.match(SimpleMultilevelArgument.java:34) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterImpl.match(AnnotatedParameterImpl.java:35) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterStateImpl.lambda$new$0(AnnotatedParameterStateImpl.java:20) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.HandleLazyUtil.lambda$handle$0(HandleLazyUtil.java:16) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.panda_std.Lazy.get(Lazy.java:50) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.AnnotatedParameterStateImpl.matchResult(AnnotatedParameterStateImpl.java:26) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteArgumentArgumentExecutor.find(LiteArgumentArgumentExecutor.java:78) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.find(LiteCommandSection.java:244) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.find(LiteCommandSection.java:225) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.implementation.LiteCommandSection.execute(LiteCommandSection.java:83) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.command.CommandService.lambda$register$0(CommandService.java:42) ~[?:?]
        at ru.elementcraft.elementcomets.__relocated__.litecommands.bukkit.SimpleCommand.execute(SimpleCommand.java:29) ~[?:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(:152) ~[guardspigot.jar:git-Paper-8359863]
        ... 14 more
```

**Comment by Rollczi** [comment]
Thanks ❤️

**Comment by Rollczi** [comment]
Fix in version 2.8.7 LiteCommands
